projects/igniteui-angular-i18n/src/i18n/expand-type.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @internal
+ * Expands object types (one level) and makes properties required
+ * https://stackoverflow.com/a/57683652
+ */
+type ExpandRequire<T> = T extends infer O ? { [K in keyof O]-?: O[K] } : never;
```

Comment:
There's a no-unused-vars warning for this (see actions output). Although that shouldn't be the case as per https://stackoverflow.com/a/63798580, it still does appear even with newer versions. Can you disable the rule here?